packages/babel-plugin-transform-es2015-template-literals/src/index.js

```diff
@@ -18,7 +18,9 @@ export default function ({ types: t }) {
         let raw = [];
 
         for (const elem of (quasi.quasis: Array)) {
-          strings.push(t.stringLiteral(elem.value.cooked));
+          if (elem.value.cooked !== null) {
```

Comment:
Not sure what's supposed to happen for `.length` of `strs` or if there is danger that one of several pieces might be missing cooked, like what is the output for `\unicode${45}\u0065`, are both pieces missing the `.cooked`?